CHANGELOG/CHANGELOG-3.5.md

```diff
@@ -334,6 +334,10 @@ See [code changes](https://github.com/etcd-io/etcd/compare/v3.4.0...v3.5.0) and
 
 **Again, before running upgrades from any previous release, please make sure to read change logs below and [v3.5 upgrade guide](https://etcd.io/docs/latest/upgrades/upgrade_3_5/).**
 
+### Improved
+
+- [print gRPC metadata in guarnteed order using the official go fmt pkg](https://github.com/etcd-io/etcd/pull/18312).
```

Comment:
Thanks for raising this @mohamedawnallah - This entry needs to be included under the `v3.5.15` section under a header `### Package clientv3`.

Reply:
Thanks! updated